pkg/statistics/handle/types/interfaces.go

```diff
@@ -418,15 +418,23 @@ type StatsSyncLoad interface {
 	HandleOneTask(sctx sessionctx.Context, lastTask *NeededItemTask, exit chan struct{}) (task *NeededItemTask, err error)
 }
 
+// GlobalStatsInfo represents the contextual information pertaining to global statistics.
+type GlobalStatsInfo struct {
+	// When the `isIndex == 0`, HistIDs will be the column IDs.
```

Comment:
Could you please move the comment to the corresponding field?

Reply:
make sense